Octokit.Reactive/Clients/Enterprise/IObservableEnterpriseAdminStatsClient.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using Octokit;
```

Comment:
I have a strange behavior in visual studio. Vsual Studio tells me that it cannot resolve typ or namespace for types that came from namespace Octokit. But that comes first since yesterday. 

![unbenannt](https://cloud.githubusercontent.com/assets/19551763/15930228/01bc6d8a-2e52-11e6-9350-304a39fc6168.PNG)